### ISSUE TITLE
Travis-ci.org shut down will occur on May 31, 2021

### ISSUE THREAD

**Issue by shogo82148** [reviewdog/reviewdog#939]
Travis CI announced the org domain will shut down.

- https://blog.travis-ci.com/2021-05-07-orgshutdown

> - Travis-ci.org shut down will occur on May 31, 2021
> - This change is to ensure product updates and fixes can be implemented for all users, at the same time
> - All your repositories on travis-ci.org should be migrated to travis-ci.com before May 31, 2021 to avoid builds not running
> - All repositories – private, and public – will continue to build as expected on travis-ci.com once migrated
> - Your historic build data on travis-ci.org will remain available in a read-only state for an indefinite period

We need some actions, because our CI on Travis CI is still on the org domain: https://travis-ci.org/github/reviewdog/reviewdog

Our choices are:

- Drop supporting Travis CI
- Migrating to the com domain and upgrade to the paid plan
  - The lowest concurrency based plan is 1 Concurrent plan: $69/month
  - ref. https://docs.travis-ci.com/user/billing-overview/#travis-ci-plan-types
- Migrating to the com domain and upgrade to the OSS plan
  - we need to contact the Travis CI support team
  - ref. https://docs.travis-ci.com/user/billing-faq/#what-if-i-am-building-open-source

@haya14busa How do you think?

**Comment by haya14busa** [comment]
Thanks for the heads up!
I just migrated the repo to travis.com with Free plan.
Probably we can drop Travis CI test in reviewdog repo, but I'd like to keep it as-is  for now as long as we have a free plan credit.

**Comment by shogo82148** [comment]
I guess you misunderstand the free plan. (also, I did)
The free plan doesn't offer 10000 Credits/month, but only replenish 10000 Credits at first time.

https://docs.travis-ci.com/user/billing-overview/#free-plan
> A ‘Free’ Plan, assigned automatically to every new sign up, is a Usage based plan with an unllimited amount of users which comes with a trial pool of credits to be used. Once these credits are used they are not replenished. Request OSS Credits allowance or please consider one of our available plans.

https://github.com/reviewdog/reviewdog/pull/941 reduces the usage of credits, but it may not be effective you expected.
Builds on Travis CI will stop eventually even if we merge it.
We should upgrade other plan while we have free plan credits.
Requesting OSS Credits allowance looks the best way.

**Comment by haya14busa** [comment]
https://docs.travis-ci.com/user/billing-faq/#what-if-i-am-building-open-source

> **Each of the Travis CI Plans contains an amount of special OSS credits per month** assigned to run builds only on public repositories. To find out more about it please contact the Travis CI support team. In the email please include:

I guess we can still get special OSS credit per month according to the doc?

I'll merge the p-r anyway and we can reconsider if it's not true.
Probably we need to estimate the amount of credits if we're going to request an OSS plan. We can estimate it with the OSS or Free plan credit.

**Comment by shogo82148** [comment]
Yes, we maybe get special OSS credits per month, because the reviewdog is one of awesome OSS projects.
However, the OSS plan differs from the Free Plan. It is required to contact the Travis CI support team.

**Comment by shogo82148** [comment]
@haya14busa Can you check the status of travis-ci.com ?
It looks that the CI on travis-ci.com is not working.

https://travis-ci.com/github/reviewdog/reviewdog/branches

![image](https://user-images.githubusercontent.com/1157344/126167357-f0b4e49c-0236-4ca0-8743-1216b08bb2fa.png)

The latest build on the master branch ran on https://github.com/reviewdog/reviewdog/commit/5f61a27ccaa57273e4904112a73fa11ef2a1d435 about a month ago.
However, the latest commit on the master branch is https://github.com/reviewdog/reviewdog/commit/02b71076ef380478602006378e4cbd27dafbcdc6 that is committed 2 days ago.
